reconcile/ocm_clusters.py

```diff
@@ -18,17 +18,50 @@
 QONTRACT_INTEGRATION = "ocm-clusters"
 
 
+def _set_rosa_ocm_attrs(cluster: Mapping[str, Any]):
+    """Cluster account (aws) attribute from App-interface differs from the OCMSpec.
+    App-interface's account includes the details for all the OCM environments
+    but the cluster only needs the target OCM environment where it belongs.
+    This method changes the cluster dicctionary to include just those.
+    """
+
+    rosa = cluster["spec"]["account"]["rosa"]
+    ocm_env = [
+        env
+        for env in rosa["ocm_environments"]
+        if env["ocm"]["name"] == cluster["ocm"]["name"]
+    ]
+
+    if len(ocm_env) != 1:
+        logging.error(
+            "ROSA OCM attributes error: Check the account configuration for ROSA {cluster_ocm_env_name} OCM env"
```

Comment:
where does this `cluster_ocm_env_name` come from? What do we have exactly to check? Can you write the error message from the point of view of somebody that may not know a lot about ROSA and OCM environments?

Reply:
Sorry, I removed the variable because it's not needed anymore. I've also improved the message.